### DISCUSSION TITLE
application not building successfully

### DISCUSSION THREAD

**Discussion by Zusykss** [wailsapp/wails#2613] [General]
Good afternoon, I'm doing a wails build file.app does not open, but if you go into it and start the terminal, it works and the application opens. Tell me what I did wrong, what happened. Code attached

`= wails.Run(&options.App{
		Title:             "anty-code",
		Width:             800,
		Height:            600,
		MinWidth:          800,
		MinHeight:         600,
		MaxWidth:          1920,
		MaxHeight:         1080,
		DisableResize:     false,
		Fullscreen:        false,
		Frameless:         false,
		StartHidden:       false,
		HideWindowOnClose: false,
		BackgroundColour:  &options.RGBA{R: 255, G: 255, B: 255, A: 255},
		Assets:            assets,
		Menu:              nil,
		Logger:            nil,
		LogLevel:          logger.DEBUG,
		OnStartup:         app.startup,
		OnDomReady:        app.domReady,
		OnBeforeClose:     app.beforeClose,
		OnShutdown:        app.shutdown,
		WindowStartState:  options.Normal,
		Bind: []interface{}{
			app,
		},
		// Windows platform specific options
		Windows: &windows.Options{
			WebviewIsTransparent: false,
			WindowIsTranslucent:  false,
			DisableWindowIcon:    false,
			// DisableFramelessWindowDecorations: false,
			WebviewUserDataPath: "",
		},
		// Mac platform specific options
		Mac: &mac.Options{
			TitleBar: &mac.TitleBar{
				TitlebarAppearsTransparent: true,
				HideTitle:                  false,
				HideTitleBar:               false,
				FullSizeContent:            false,
				UseToolbar:                 false,
				HideToolbarSeparator:       true,
			},
			Appearance:           mac.NSAppearanceNameDarkAqua,
			WebviewIsTransparent: true,
			WindowIsTranslucent:  true,
			About: &mac.AboutInfo{
				Title:   "anty-code",
				Message: "",
				Icon:    icon,
			},
		},
	})`

**Comment by misitebao** [comment]
Please use [Issue](https://github.com/wailsapp/wails/issues/new?assignees=&labels=Bug&template=bug_report.yml) to provide reproducible code and errors.
